test/basic.test.js

```diff
@@ -344,3 +344,26 @@ test('render a single quote as JSON', (t) => {
   t.equal(output, JSON.stringify(toStringify))
   t.ok(validate(JSON.parse(output)), 'valid schema')
 })
+
+test('should not break when the value of field is null', (t) => {
+  t.plan(2)
+
+  const schema = {
+    title: 'an object in a string',
+    type: 'object',
+    properties: {
+      updatedAt: {
+        type: ['string'],
+        format: 'date-time'
```

Comment:
Perhaps we don't specify any date formats in this file and let `date.test.js` takes care of those cases for better organization.